scripts/install-windows-prereqs.ps1

```diff
@@ -602,6 +602,11 @@ function Install-DCAP-Dependencies {
                 }
                 Write-Output $install
             }
+            elseif (($LaunchConfiguration -eq "SGX1FLC-NoDriver") -and (${OS_VERSION} -eq "WinServer2016"))
```

Comment:
The PR description says to do this for non-SGX but this configuration is for an SGX machine, just with no driver. Is this intentional?

Reply:
I am running a regression with these lines reverted here:

https://oe-jenkins-tf.westeurope.cloudapp.azure.com/job/CI-CD_Infrastructure/job/OpenEnclave-E2E-Testing/181/

Reply:
@jhand2 

The `SGX1FLC-NoDriver` with `WinServer2016` is what we use for our machines in CI that run Windows simulation tests.

To be more specific, those are the Win Server 2016 machines what we don't install SGX driver on. And the SGX driver is not automatically setup via Windows Update on Win2016, so it doesn't matter at the end of the day if it has SGX or not since it's not used.

On these Windows Server 2016 machines, we run Windows simulation tests. And, the DLLs needs to be into system PATH. Otherwise, from what I know, the tests will fail.

However, this DLL copy is not needed on 2019, and that's why it was completely removed here: https://github.com/openenclave/openenclave/commit/0d74e2202fe8effce89aa635b6829f6875ce0b10#diff-846bcbf4c48fa80e76202a5e31d77476